transcom/mymove
# B 20619 prime create mobile home int

## [B-20619](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A982505)

## Summary

This PR adds the capability for Prime counselors to create mobile home shipments for existing moves.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Login as Prime
2. Choose any move
3. Click "Create Shipment" button and choose "MOBILE_HOME"
4. Confirm unique mobile home shipment fields show
5. Confirm to see mobile home shipment success banner 
6. From the confirmation banner you can use that id for this query to check in DB that it was added
`select id, shipment_type, status  from mto_shipments ms where id ='shipment id'`

### Frontend


### Backend


## Screenshots
**_Mobile Home Form_**
<img width="1072" alt="Screenshot 2024-09-13 at 1 53 53 PM" src="https://github.com/user-attachments/assets/c01153ae-da82-4237-b078-22e4ac39b6a2">

**_Mobile Home added to shipment types dropdown list_**
<img width="910" alt="Screenshot 2024-09-13 at 1 53 25 PM" src="https://github.com/user-attachments/assets/91db7211-6582-4f0f-805e-82f58b845316">

**_Created Mobile Home Shipment Successful banner_**
<img width="1141" alt="Screenshot 2024-09-13 at 1 51 05 PM" src="https://github.com/user-attachments/assets/3b726358-ac82-4654-850a-cbce76a6c2ff">

**_Mobile Home Shipment specific fields_**
<img width="953" alt="Screenshot 2024-09-17 at 2 10 49 PM" src="https://github.com/user-attachments/assets/862ca8a6-779d-44c9-999f-4a287cad18e9">
